packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -219,13 +220,15 @@ async function searchLatestTag(): Promise<void> {
       image = image.slice(DOCKER_PREFIX_WITH_SLASH.length);
     }
     const tags = await window.listImageTagsInRegistry({ image });
+    isValidName = Boolean(tags);
```

Comment:
no tests ?

Reply:
Tests should for this are in https://github.com/podman-desktop/podman-desktop/pull/9782

Reply:
well, there is a test of TypeAhead, but we should still have tests there to check that depending on the response from listImageTagsInRegistry, isValidName can be either false or true and then the error attribute

Reply:
TypeAhead unit test is to check that if there is error, the colors are correctly changed

but here, unit test is to check that isValidName is correctly updated based on tags result